server.py

```
@@ -342,195 +549,341 @@ async def create_time_entry(
+                   Defaults to current week's Monday.
+    """
+    if not start_date:
+        # Get current week's Monday
+        today = datetime.now()
+        start = today - timedelta(days=today.weekday())
+        start_date = start.strftime("%Y-%m-%d")
     
     try:
-        today = datetime.now().strftime("%Y-%m-%d")
+        api_token = get_api_token()
+        client = TimeCampClient(api_token)
```

Ruff (0.11.9)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)